ivy/functional/frontends/torch/tensor.py

```diff
@@ -988,6 +988,11 @@ def count_nonzero(self, dim):
     def exp(self):
         return torch_frontend.exp(self)
 
+    @with_unsupported_dtypes({"1.11.0 and below": ("int8", "int16", "int32", "int64", "uint8", "bool", "float16")}, "torch")
```

Comment:
```suggestion
    @with_unsupported_dtypes({"1.11.0 and below": ("int8", "int16", "int32", "int64", "uint8", "bool", "float16")}, "torch") #  noqa
```

Be sure to enable `pre-commit` when committing this change, `black` may attempt to fix some whitespaces that I may have missed

Reply:
Well, i did add #noqa but black during pre-commit will change the format. So, i disabled the pre-commit.

Reply:
and removed the #noqa.

Reply:
Any Suggestions?

Reply:
I found this method on the stackoverflow:
```
# fmt: off
your code
# fmt: on
```

This will disable black on the lines where fmt was off and will enable for the lines starting from fmt: on.

Reply:
ok, let me do the commit with `# noqa`.